core/templates/expressions/expression-syntax-tree.service.ts

```diff
@@ -67,9 +67,9 @@ export class ExprWrongArgTypeError extends ExpressionError {
   constructor(
       public arg: number|string,
       public actualType: string, public expectedType: string) {
-    super(
-      (arg !== null ?
-        (arg + ' has type ' + actualType + ' which') : ('Type ' + actualType)) +
+    super((
+      arg !== null ?
+      (arg + ' has type ' + actualType + ' which') : ('Type ' + actualType)) +
       ' does not match expected type ' + expectedType);
```

Comment:
```suggestion
    super(arg !== null ? (arg + ' has type ' + actualType + ' which') : (
        'Type ' + actualType does not match expected type ' + expectedType);
```

This looks much better? wdyt?

Reply:
Changed to the suggestion